### PR TITLE
feat: replace color picker

### LINKED ISSUE #2908: Use a different color picker
**Describe the feature you'd like**
The color picker used currently should be replaced, as it's maven dependency is not accepted in F-droid

Parent issue: #2907 

**Would you like to work on the issue?**

- [x] Yes
- [ ] No
- Other:

### PR DESCRIPTION
**Pull request by codedsun**
Fixed #2908 

Changes: Used a new library for f-droid inclusion

Screenshots of the change:

### CODE REVIEW

**Comment by codedsun** [comment]
@yashk2000 Please check

**Comment by yashk2000** [comment]
Also, the error you were facing while building the app for fdroid with the old color picker, gets resolved when you use this one?

**Comment by codedsun** [comment]
@yashk2000 Please check the issue #2907. 
>Could you please provide a screenshot of how it looks right now?
Yes i will provide

**Comment by codedsun** [comment]
ezgif com-video-to-gif

@yashk2000 Check

**Comment by yashk2000** [comment]
> @yashk2000 Please check the issue #2907.

That's what I was asking. Using this color picker resolves the fdroid issue?

**Comment by codedsun** [comment]
@yashk2000 this is not needed as the master branch had some dependencies in v1.9.0 which were removed in development and while inclusion for f-droid they pointed some maven dependencies to remove, which caused the color picker to replaced with some other dependency which doesn't come from maven.

Closing this issue, as not needed.

**Comment by codedsun** [comment]
@iamareebjamal Reopening this PR as we have to get this done for fdroid inclusion

>ERROR: Found unknown maven repo 'https://dl.bintray.com/dasar/maven' at build.gradle

The old color picker uses this repo which is not allowed in f-droid

**Comment by codedsun** [comment]
@yashk2000 Please review this!

**Comment by codedsun** [comment]
@iamareebjamal This need to be merged as for the fdroid inclusion.

Ref: https://gitlab.com/fdroid/fdroiddata/merge_requests/5877#note_261889667

**Comment by iamareebjamal** [comment]
Remove the maven repo as well or there is no point in replacing the library

**Comment by codedsun** [comment]
@iamareebjamal Just a sec, updating

File: build.gradle [hunk 1]
```diff
@@ -30,7 +30,6 @@ allprojects {
         jcenter()
         mavenCentral()
         maven { url "https://repo1.maven.org/maven2" }
```
Comment (iamareebjamal) [inline build.gradle]: Remove this as well. Why do we need this?

Comment (codedsun) [reply build.gradle]: I didn't know why it was been used, not used anywhere. I tested the app working fine with removing this dependency.

**Pull request merged**